packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -182,1084 +183,2282 @@ describe('Profiler', () => {
     });
   });
 
-  [true, false].forEach(enableSchedulerTracing => {
-    describe('onRender callback', () => {
-      beforeEach(() => {
-        jest.resetModules();
+  [true, false].forEach(deferPassiveEffectCleanupDuringUnmount => {
```

Comment:
This method of testing feature flags is controversial. I think it's nice to have the extra coverage, and these tests are already very verbose so the idea of duplicating them is not very appealing- but I don't want to die on this hill. I'll remove this and just test one of the flag settings if that's the team preference.